### PR TITLE
Automate knativekafka manifest update in hack/update-manifests.sh, update knativekafka to v0.21.1

### PR DESCRIPTION
**Pull request by matzew**
This is an updated version of https://github.com/openshift-knative/serverless-operator/pull/831

changed the location of the download/patching for the kafka bits to be in the `knative-operator`

### CODE REVIEW

**Comment by matzew** [comment]
/hold
waiting for reviews from Lukas/Ali

**Comment by matzew** [comment]
```
 Error from server (NotFound): error when replacing "STDIN": configmaps "config-tracing" not found 
INFO[2021-05-03T16:20:29Z] 16:19:09.095 ERROR:   🚨 Error (code: 1) occurred at /go/src/knative.dev/eventing-kafka/openshift/e2e-common.sh:175, with command: oc replace -f - 
INFO[2021-05-03T16:20:29Z] 16:19:09.098 INFO:    Dumping state...       
INFO[2021-05-03T16:20:29Z] 16:19:09.100 DEBUG:   Environment variables: 
INFO[2021-05-03T16:20:29Z] PROW_JOB_ID=1cf6d8bd-ac21-11eb-ae6f-0a580a831230 
INFO[2021-05-03T16:20:29Z] TEST_KNATIVE_E2E=true                        

```


hrm...

File: openshift-knative-operator/hack/update-manifests.sh [hunk 1]
```diff
@@ -41,28 +41,6 @@ function download {
   done
 }
 
-function download_ingress {
```
Comment (markusthoemmes) [inline openshift-knative-operator/hack/update-manifests.sh:44]: I'm guessing this ain't intentional?

Comment (matzew) [reply openshift-knative-operator/hack/update-manifests.sh:44]: yes, that's a rebase issue - I anm aware of that ;-)

**Pull request merged**